### PR TITLE
Working fork

### PR DESCRIPTION
**Pull request by andrioid**
Hi. I needed to import Pinboard posts to my blog, so I started using your plugin.

There was something missing, so I added it for my purposes. You can merge it, if you want. I can also adapt it, if you have any comments.

### CODE REVIEW

File: gatsby-node.js
Comment (jondubin) [inline gatsby-node.js]: ✂️

File: gatsby-node.js
Comment (jondubin) [inline gatsby-node.js]: ✂️

File: gatsby-node.js
Comment (jondubin) [inline gatsby-node.js]: ✂️

File: gatsby-node.js
Comment (jondubin) [inline gatsby-node.js]: ✂️

File: gatsby-node.js
Comment (jondubin) [inline gatsby-node.js]: ✂️

File: package.json
Comment (jondubin) [inline package.json]: Wanna change this back to `gatsby-source-rss` so that it's the "official", unforked version?

File: src/gatsby-node.js
Comment (jondubin) [inline src/gatsby-node.js]: ✂️

File: src/gatsby-node.js
Comment (jondubin) [inline src/gatsby-node.js]: ✂️

File: src/gatsby-node.js
Comment (jondubin) [inline src/gatsby-node.js]: ✂️

File: src/gatsby-node.js
Comment (jondubin) [inline src/gatsby-node.js]: ✂️

**Review by jondubin — COMMENTED** [review]
Hey! Looks awesome. Thank you for taking the time to work on this. Just left some comments around removing dead code, but otherwise LGTM

File: package.json
Comment (andrioid) [reply package.json]: Of course.

**Comment by andrioid** [comment]
I've removed the comments and the console.logs. So this should be in a semi-good shape. Let me know if there are other concerns.

**Comment by andrioid** [comment]
@jondubin Ready to merge and publish?

**Review by jondubin — APPROVED** [review]

**Comment by jondubin** [comment]
Yep, feel free to merge it in!

**Comment by andrioid** [comment]
I can't merge it. Github says "Only those with 'write access' to this repository can merge pull requests"

**Comment by andrioid** [comment]
Can you bump the version and publish to npm or give me access?

**Pull request merged**
